guarddog/analyzer/sourcecode/exfiltrate-sensitive-data.yml

```diff
@@ -42,6 +55,7 @@ rules:
               - pattern-inside: $S = socket.socket(...); ...
               - pattern-inside: $S.connect(...); ...
               - pattern-inside: $S.send(...)
+          - pattern-inside: $F.write(...)
```

Comment:
the rule aims to detect an exfiltration, writing to a file i think it does not qualifies as exfil. thoughts?

Reply:
I was sort of hesitant about this, as well.  I think you're right, I will change the test case and leave the sinks alone.